File: service-java/code/src/main/java/{{package_name.value}}/ObjectMapperConfig.java
package {{package_name.value}};

import com.fasterxml.jackson.databind.ObjectMapper;
import org.springframework.context.annotation.*;
import {{package_name.value}}.json.Json;

public class ObjectMapperConfig {
	@Bean
	@Primary
	public ObjectMapper objectMapper() {
		ObjectMapper objectMapper = new ObjectMapper();
		Json.setupObjectMapper(objectMapper);
		return objectMapper;
	}
}

File: service-kotlin/{{#server.spring}}/src/main/kotlin/{{package.value}}/{{#jsonlib.jackson}}/ObjectMapperConfig.kt
package {{package.value}}

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
import org.springframework.context.annotation.*
import {{package.value}}.json.setupObjectMapper

@Configuration
class ObjectMapperConfig {
    @Bean
    @Primary
    fun objectMapper(): ObjectMapper {
        val objectMapper = jacksonObjectMapper()
        setupObjectMapper(objectMapper)
        return objectMapper
    }
}